docs/quadratic/quadratic funding.md

```diff
@@ -0,0 +1,69 @@
+# What Is Quadratic Funding?
+
+Quadratic Funding is a mechanism for allocating funds in a decentralized and democratic manner. It is designed to support public goods, such as open-source software, research, and community initiatives, by leveraging contributions from both large and small donors. The key idea behind Quadratic Funding is to match donations to projects not linearly, but quadratically, amplifying the impact of smaller contributions. [To learn more](https://www.wtfisqf.com/)
```

Comment:
`To learn more` feels bit out of context. Providing users with additional context about the destination of the documentation link can be beneficial. For example
```suggestion
Quadratic Funding is a mechanism for allocating funds in a decentralized and democratic manner. It is designed to support public goods, such as open-source software, research, and community initiatives, by leveraging contributions from both large and small donors. The key idea behind Quadratic Funding is to match donations to projects not linearly, but quadratically, amplifying the impact of smaller contributions. Check out [Wtfisqf](https://www.wtfisqf.com/) for in depth explanation.
```